client/state/plugins/installed/selectors.js

```diff
@@ -0,0 +1,164 @@
+import filter from 'lodash/filter';
+import forEach from 'lodash/forEach';
+import values from 'lodash/values';
+import sortBy from 'lodash/sortBy';
+import find from 'lodash/find';
+import uniq from 'lodash/uniq';
+import compact from 'lodash/compact';
+import omit from 'lodash/omit';
+import every from 'lodash/every';
+import some from 'lodash/some';
+
+const _filters = {
+	none: function() {
+		return false;
+	},
+	all: function() {
+		return true;
+	},
+	active: function( plugin ) {
+		return plugin.sites.some( function( site ) {
+			return site.plugin && site.plugin.active;
+		} );
+	},
+	inactive: function( plugin ) {
+		return plugin.sites.some( function( site ) {
+			return site.plugin && ! site.plugin.active;
+		} );
+	},
+	updates: function( plugin ) {
+		return plugin.sites.some( function( site ) {
+			return site.plugin && site.plugin.update && site.canUpdateFiles;
+		} );
+	},
+	isEqual: function( pluginSlug, plugin ) {
+		return plugin.slug === pluginSlug;
+	}
+};
+
+const isRequesting = function( state, siteId ) {
+	// if the `isRequesting` attribute doesn't exist yet,
+	// we assume we are still launching the fetch action, so it's true
```

Comment:
yeah, it's already changed locally, I just didn't push last night since I'm still working through other feedback.